Add files via upload

jm14301 opened the pull request:
I added Categories and Shop by Brand pages, and a couple of company logos

jm14301 committed:
Add files via upload

daliadelgadillo reviewed (changes requested):
Change categories to circle images (discussed target app example)
change shop by brand from dropdown to product carousels (see homepage)
- Wait for all products to be picked out by whole group, sort those products into the different brands

jm14301 committed:
Categories

- Made changes to Shop By Brand Pages
- Added one class to style.css
- Images added
